### PR TITLE
Interactive to do list

### PR DESCRIPTION
**Pull request by Umuhan**
### Below are the changes appied to my code:
- [x] Added a new JavaScript file and imported it as a module:
- [x] Added event listener to the checkbox .
- [x] Implemented a function for the "Clear all completed" button (using filter() method).
- [x] Stored the updated array of items in local storage

### CODE REVIEW

File: src/modules/localStorage.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Function to save tasks to local storage
+function saveTasksToLocalStorage(tasks) {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
```
Comment (OyePriscilla) [inline src/modules/localStorage.js:4]: - [ ] Hello, I observed that the todos were only displayed in the UI; kindly ensure that the array of todos is stored in the local storage as well. Below is a screenshot 👇 

Okota flyer-modified (6)

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Umuhan 👍, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Add event listener to the checkbox ✔️ 
Implemented a function for the "Clear all completed" button ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/localStorage.js (hunk 1 above)
Comment (Umuhan) [reply src/modules/localStorage.js:4]: done

**Review by Alusp — APPROVED** [review]
Hi @Umuhan, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congrats

## Highlights

- [ ] Add event listener to the checkbox ✔️
- [ ] Implemented a function for the "Clear all completed" button ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alusp in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**